ui/src/app/machines/views/MachineDetails/MachineSummary/OverviewCard/_index.scss

```diff
@@ -0,0 +1,256 @@
+@mixin pseudo-border($pos) {
```

Comment:
This scss file is really not great. I'll definitely want to rewrite this once the overview card is finished content-wise, so I wouldn't worry too much about the styling.